### ISSUE TITLE
[BUG] `defaultLanguage` is required

### ISSUE THREAD

**Issue by fmacherey** [Usercentrics/react-native-sdk#7]
**Describe the bug**
It seems that `defaultLanguage` from `new UsercentricsOptions("id")` is required. When I omit this, our App crashes with an SIGABRT in Xcode. I logged the returned default options and defaultLanguage is set to `undefined`. 

Solution: 
a) extend docs and example to be required, or
b) define fallback language for default options.

I think a) might be a better approach, but this is up to you :) 

**usercentrics/react-native branch/commit**
- v0.0.3

**Comment by uc-pedro** [comment]
Hello @fmacherey thanks for opening this issue. 

Unfortunately I could not reproduce it, since the only required field to initialize our SDK is the settingsId. 
Can you try using this snippet `Usercentrics.configure({settingsId: "SettingsId"});`?

Thank you!

**Comment by fmacherey** [comment]
Hey @uc-pedro 

thanks, that worked. But we are using the variant with options, as we want to set the language. When testing the options, we found out, that the language switch is not working. We have german (default lang), dutch and english, but always get the german CMP popup. I used "en", "english" or "English" as values, but every time the popup is still in german. 

We use our locale from react-intl package, which is "de", "nl" or "en", depending on system language of the device.

Should I open a new issue, or rename this one?
